frontend/.env.production

```diff
@@ -2,3 +2,5 @@
 # This file is checked into your git repo and provides defaults for non-sensitive
 # env vars when running `next dev` only.
 # Learn more: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+
+ENVIRONMENT=prod
```

Comment:
Forgot this file existed, and don't think it needs to exist. Let's maybe talk through in a frontend sync, but in any case I don't think we need to make this update now